src/control/cmd/daos_server/storage_scm_test.go

```diff
@@ -265,24 +421,150 @@ func TestDaosServer_resetPMem(t *testing.T) {
 			},
 			expErr: errors.New("unexpected state"),
 		},
+		"sock specified in command": {
+			cmdSockID: &one,
+			expCalls: []*storage.ScmPrepareRequest{
+				{
+					Reset:    true,
+					SocketID: &one,
+				},
+			},
+			// Only checking that request was made with provided socket so expect
+			// failure.
+			expErr: errors.New("unexpected state"),
```

Comment:
These expected errors for normal situations are a bit confusing for the test. Should these be broken out into a separate test, maybe? Test a lower level helper function instead?

Reply:
done